### PR TITLE
PR Template: add optional details section

### PR DESCRIPTION
**Pull request by calebroseland**
## Summary
Amends the PR template to include an optional Details section with copiable examples.

## Ticket Link
None

## Checklist

- [ ] Telemetry updated
- [ ] Gated by experimental feature flag
- [ ] Unit tests updated

### CODE REVIEW

**Review by trilopin — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,14 +12,29 @@ REMEMBER TO:
...
-A description of what this pull request does
+A short description of what this pull request does
+-->
+
+<!-- 
+Describe changes in more detail when additional clarity is needed. Consider adding screenshots or describing each commit.
+## Details
+
+- d77ea0dd3491882e8ba3d25bea6017232a87385b: adds a thing
+- d77ea0dd3491882e8ba3d25bea6017232a87385b: changes this other thing
```
Comment (lieut-data) [inline .github/PULL_REQUEST_TEMPLATE.md:23]: nit: each commit should already be self-described in the commits tab, right?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -12,14 +12,29 @@ REMEMBER TO:
 
 ## Summary
 <!--
-A description of what this pull request does
+A short description of what this pull request does
+-->
+
+<!-- 
+Describe changes in more detail when additional clarity is needed. Consider adding screenshots or describing each commit.
+## Details
```
Comment (lieut-data) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: Is the title then already functioning as the summary? I've almost always jumped into the description section from the first paragraph, but just aimed to "keep it short".

**Review by lieut-data — CHANGES_REQUESTED** [review]
I'm supportive of the general direction, but worried about making things too complicated. I already spend time deleting all of this when pushing up my PRs and reformatting manually, but of course see the value in helping first-time committers. Some thoughts inline.

**Comment by lieut-data** [comment]
Closing this out for now.

**Pull request closed without merging**